site/content/internal/release-process.md

```diff
@@ -6,9 +6,13 @@ section: internal
 
 # Release Cutting Process
 
-**Note: To cut a build you need access to matterbuild. Please ask Christopher for access if you don't have it.**
+**Note: To cut a build you need access to matterbuild. Please ask Christopher/Carlos for access if you don't have it.**
 
-On code complete day, work with the PM on rotation to get all the pull requests for the current release merged into `master`. Once that is done and you've confirmed with the PM, cut the first release candidate by following these steps:
+Developers and PMs decides when to create the release branch. The branch can be created from master or from a existing release branch, it depends for which type of release we are planning. If is a Quality Release the new branch should be branch out from a previous release, otherwise the branch should happen from the master branch.
```

Comment:
```suggestion
Developers and PMs decide when to create the release branch. The branch can be created from master or from an existing release branch, it depends which type of release we are planning. If it is a Quality Release, the new branch should be branched off the previous release, otherwise the branch should be off the master branch.
```

Minor grammar suggesstions